__tests__/index.ts

```diff
@@ -551,6 +551,23 @@ describe('Mongo', () => {
     });
   });
 
+  describe('unsetService', () => {
+    it('should not convert id', async () => {
```

Comment:
Where's the `expect` in this `it` block?

Reply:
there is no expect because it should throw is the test is failing

Reply:
Shouldn't be expect.toThrow?

Reply:
It should not throw, it will throw if the function was converting the id

Reply:
not sure I'm getting the point then.. sorry

Reply:
So the function check that if you have the option convertUserIdToMongoObjectId if you pass a non valid mongo string it should not throw. If the option was not handled properly the function should throw.
Do you think about a better way to test it?

Reply:
mmm either mock the update func and expect the same user id you passed will be the best option I think

Reply:
Thanks for explaining the issue though. I missed that on first read..

Reply:
Fixed :)